composer.json

```diff
@@ -6,20 +6,18 @@
   "license": "GPL-2.0-or-later",
   "minimum-stability": "dev",
   "require": {
-    "drupal/layout_paragraphs": "^1.0@beta",
-    "drupal/paragraphs": "^1.11"
+    "drupal/layout_paragraphs": "dev-1.0.x",
+    "drupal/paragraphs": "^1.11",
```

Comment:
This is a dependency of both localgovdrupal/localgov_paragraphs and drupal/layout_paragraphs, so not needed here.

Reply:
Hi Steve, I agree drupal/paragraphs should not be here.  Not that I added it :)